rpm/sdk-setup.spec

```diff
@@ -225,6 +226,10 @@ chmod 755 %{buildroot}%{_bindir}/*
 
 %pre -n sdk-chroot
 if ! rpm --quiet -q ca-certificates && [ -d /%{_sysconfdir}/ssl/certs ] ; then echo "Cleaning up copied ssl certs. ca-certificates should now install"; rm -rf /%{_sysconfdir}/ssl/certs ;fi
+rm -Rf /home/.zypp-cache
```

Comment:
If we decide later to clean this up where it was created, it might go away. Should this check for existence before removing?